mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -2033,20 +2033,71 @@ static Value extractInsertFoldConstantOp(OpType op, AdaptorType adaptor,
 static Attribute foldPoisonIndexInsertExtractOp(MLIRContext *context,
                                                 ArrayRef<int64_t> staticPos,
                                                 int64_t poisonVal) {
-  if (!llvm::is_contained(staticPos, poisonVal))
+  if (!is_contained(staticPos, poisonVal))
```

Comment:
nit: I'd undo this so that we don't depend on [ADL](https://en.cppreference.com/w/cpp/language/adl).
Unlike the cast functions and container types like SmallVector/StringRef/ArrayRef/etc, we don't have a using declaration in the mlir namespace for functions from STLExtras.h.